src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.figureType = "square";
```

Comment:
I think this field is redundant. You are using it in the `toString` method. You can remove that field and just type that string in `toString` method.